README.md

```diff
@@ -44,6 +44,15 @@ Or force a chef run:
 
 This subcommand operates in a manner similar to [knife ssh](https://docs.chef.io/knife_ssh.html)...just leveraging the WinRM protocol for communication. It also include's `knife ssh`'s "[interactive session mode](https://docs.chef.io/knife_ssh.html#options)"
 
+### knife wsman test
+
+Connects to the remote WSMan/WinRM endpoint and verifies the remote node is listening.  This is the equivalent of running Test-Wsman from PowerShell.  Endpoints to test can be specified manually, or be driven by search and use many of the same connection options as knife winrm.
+
+    knife wsman test 192.168.1.10 -m
```

Comment:
So --help prints a lot, the bulk of which isn't relevant most of the time, but maybe is periodically relevant.

An example like this would imply some good common flags:
```
 knife wsman test --manual-list 192.168.1.10 --winrm-port 443 --winrm-transport ssl
```

---
```knife wsman test --help```  output:
```
knife wsman test QUERY (options)
    -a, --attribute ATTR             The attribute to use for opening the connection - default is fqdn
    -f CA_TRUST_FILE,                The Certificate Authority (CA) trust file used for SSL transport
        --ca-trust-file
    -s, --server-url URL             Chef Server URL
        --chef-zero-host HOST        Host to start chef-zero on
        --chef-zero-port PORT        Port to start chef-zero on
    -k, --key KEY                    API Client Key
        --[no-]color                 Use colored output, defaults to false on Windows, true otherwise
    -c, --config CONFIG              The configuration file to use
        --defaults                   Accept default values for all questions
    -d, --disable-editing            Do not open EDITOR, just accept the data as is
    -e, --editor EDITOR              Set the editor to use for interactive commands
    -E, --environment ENVIRONMENT    Set the Chef environment (except for in searches, where this will be flagrantly ignored)
    -F, --format FORMAT              Which format to use for output
        --identity-file IDENTITY_FILE
                                     The SSH identity file used for authentication
    -i, --keytab-file KEYTAB_FILE    The Kerberos keytab file used for authentication
    -R KERBEROS_REALM,               The Kerberos realm used for authentication
        --kerberos-realm
    -S KERBEROS_SERVICE,             The Kerberos service used for authentication
        --kerberos-service
    -z, --local-mode                 Point knife commands at local repository instead of server
    -m, --manual-list                QUERY is a space separated list of servers
    -u, --user USER                  API Client Username
        --print-after                Show the data after a destructive operation
        --session-timeout Minutes    The timeout for the client for the maximum length of the WinRM session
    -V, --verbose                    More verbose output. Use twice for max verbosity
    -v, --version                    Show chef version
        --winrm-authentication-protocol AUTHENTICATION_PROTOCOL
                                     The authentication protocol used during WinRM communication. The supported protocols are basic,negotiate,kerberos. Default is 'negotiate'.
    -P, --winrm-password PASSWORD    The WinRM password
    -p, --winrm-port PORT            The WinRM port, by default this is '5985' for 'plaintext' and '5986' for 'ssl' winrm transport
        --winrm-ssl-verify-mode SSL_VERIFY_MODE
                                     The WinRM peer verification mode. Valid choices are [verify_peer, verify_none]
    -t, --winrm-transport TRANSPORT  The WinRM transport type.  valid choices are [ssl, plaintext]
    -x, --winrm-user USERNAME        The WinRM username
    -y, --yes                        Say yes to all prompts for confirmation
    -h, --help                       Show this message
```